crates/runtime/src/dataaccelerator/metadata.rs

```diff
@@ -27,6 +28,11 @@ use crate::component::dataset::{acceleration::Engine, Dataset};
 pub const METADATA_TABLE_NAME: &str = "spice_sys_metadata";
 pub const METADATA_DATASET_COLUMN: &str = "dataset";
 pub const METADATA_METADATA_COLUMN: &str = "metadata";
```

Comment:
It's weird this is called `metadata`. Isn't the entire table metadata? Maybe just `data` or `value` so you have `key` + `data` | `value`. Basically, this is a K/V store for each accelerator.